src/data/layer-2/layer-2.ts

```diff
@@ -83,6 +84,22 @@ export const layer2Data: Rollups = {
       image: BobaImage,
       background: "black",
     },
+    {
+      name: "Base",
+      website: "https://base.org/",
+      developerDocs: "https://docs.base.org/",
+      l2beat: "https://l2beat.com/scaling/projects/base",
+      bridge: "https://bridge.base.org/deposit",
+      bridgeWallets: ["MetaMask", "WalletConnect", "Coinbase Wallet"],
+      blockExplorer: "https://basescan.org/",
+      ecosystemPortal: "https://www.ethereum-ecosystem.com/blockchains/base",
```

Comment:
```suggestion
      ecosystemPortal: "https://www.base.org/ecosystem",
```

Thanks @Maxservais, only change is for ecosystemPortal, we should use their ecosystem page. 

We can look at how to add ethereum-ecosystem.com in https://github.com/ethereum/ethereum-org-website/issues/11700 though